src/mol-plugin/config.ts

```diff
@@ -55,6 +55,7 @@ export const PluginConfig = {
     Viewport: {
         ShowExpand: item('viewer.show-expand-button', true),
         ShowControls: item('viewer.show-controls-button', true),
+        ShowSecondaryStructureSequence: item('viere.show-ss-sequence-button', true),
```

Comment:
- there is a typo: `viere`
- Also, with moving the toggle to a different place, I don't think we need this config anymore?